src/platform/packages/shared/kbn-analytics/src/performance/performance_tracker.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { v4 as uuidv4 } from 'uuid';
+import { groupBy } from 'lodash';
+
+export const PERFORMANCE_TRACKER_MARKS = {
+  PRE_RENDER: 'preRender',
+  RENDER_START: 'renderStart',
+  RENDER_COMPLETE: 'renderComplete',
```

Comment:
Can you please add a jsdoc describing each mark?

Reply:
Added jsdoc in [0c24d07](https://github.com/elastic/kibana/pull/216330/commits/0c24d0766e3ef54ad7efb8828e47d8d3610ecc9a).